<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qMRMLPlotViewControllerWidget</class>
 <widget class="ctkPopupWidget" name="qMRMLPlotViewControllerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>118</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plot View Controller</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <layout class="QGridLayout" name="contentLayout">
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="hboxLayout">
        <item>
          <widget class="qMRMLNodeComboBox" name="plotChartComboBox">
           <property name="toolTip">
            <string>Select the PlotChartNode which handles the general Properties of the Plot and allow to select multiple PlotSeriesNodes.</string>
           </property>
           <property name="nodeTypes">
            <stringlist>
             <string>vtkMRMLPlotChartNode</string>
            </stringlist>
           </property>
           <property name="noneEnabled">
            <bool>false</bool>
           </property>
           <property name="addEnabled">
            <bool>true</bool>
           </property>
           <property name="removeEnabled">
            <bool>true</bool>
           </property>
           <property name="renameEnabled">
            <bool>true</bool>
           </property>
           <property name="editEnabled">
            <bool>true</bool>
           </property>
          </widget>
        </item>
        <item>
          <widget class="QPushButton" name="exportPushButton">
           <property name="toolTip">
            <string>Export plot to SVG file</string>
           </property>
           <property name="text">
            <string>Export...</string>
           </property>
          </widget>
        </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="qMRMLCheckableNodeComboBox" name="plotSeriesComboBox">
       <property name="toolTip">
        <string>Add/Remove plots data series to/from the current chart.</string>
       </property>
       <property name="nodeTypes">
        <stringlist>
         <string>vtkMRMLPlotSeriesNode</string>
        </stringlist>
       </property>
       <property name="noneEnabled">
        <bool>false</bool>
       </property>
       <property name="addEnabled">
        <bool>true</bool>
       </property>
       <property name="removeEnabled">
        <bool>true</bool>
       </property>
       <property name="editEnabled">
        <bool>false</bool>
       </property>
       <property name="renameEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Plot chart:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Plot data series:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Plot type:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="plotTypeComboBox">
       <property name="toolTip">
        <string>This combobox allows to change the Type for all the active PlotSeriesNodes. If a value is chosen, all the PlotSeriesNodes referenced by the PlotChartNode will be updated with the new value. </string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>line</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>bar</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>scatter</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>scatter bar</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Interaction mode:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="interactionModeComboBox">
       <property name="toolTip">
        <string>Action performed on mouse left-click and drag.</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>pan view</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>select points</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>free-hand select points</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>move points</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionShow_Grid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Grid</string>
   </property>
   <property name="toolTip">
    <string>Show grid</string>
   </property>
  </action>
  <action name="actionShow_Legend">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Legend</string>
   </property>
   <property name="toolTip">
    <string>Show legend</string>
   </property>
  </action>
  <action name="actionFit_to_window">
   <property name="icon">
    <iconset resource="../qMRMLWidgets.qrc">
     <normaloff>:/Icons/SlicesFitToWindow.png</normaloff>:/Icons/SlicesFitToWindow.png</iconset>
   </property>
   <property name="text">
    <string>Fit_to_window</string>
   </property>
   <property name="toolTip">
    <string>Adjust the Plot Viewer's field of view to match the extent of the Plot axes</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLCheckableNodeComboBox</class>
   <extends>qMRMLNodeComboBox</extends>
   <header>qMRMLCheckableNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkPopupWidget</class>
   <extends>QFrame</extends>
   <header>ctkPopupWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qMRMLWidgets.qrc"/>
 </resources>
 <connections/>
</ui>
